src/applications/disability-benefits/2346/config/form.js

```diff
@@ -10,10 +10,14 @@ import PersonalInfoBox from '../components/PersonalInfoBox';
 import { schemaFields } from '../constants';
 import ConfirmationPage from '../containers/ConfirmationPage';
 import frontEndSchema from '../schemas/2346-schema.json';
-import { buildAddressSchema } from '../schemas/address-schema';
-import UIDefinitions from '../schemas/definitions/2346UI';
+import UIDefinitions from '../schemas/2346UI';
 
-const { email, date, supplies } = fullSchema.definitions;
+const {
+  email,
+  date,
+  supplies,
+  addressWithIsMilitaryBase,
```

Comment:
What is this trying to convey? addressWith ?

Reply:
This is the name of the address property on vets-json-schema.